engine/graphics/src/dmsdk/graphics/graphics.h

```diff
@@ -221,6 +251,45 @@ namespace dmGraphics
         TEXTURE_FORMAT_COUNT
     };
 
+    struct PipelineState
```

Comment:
Perhaps this doesn't need to live in the dmsdk? we could potentially do:
typedef struct PipelineState* HPipelineState

Reply:
It makes sense I think, as I think this struct may be a bit volatile.